include/xeus-python/xinterpreter.hpp

```diff
@@ -77,12 +77,15 @@ namespace xpyt
 
         py::object m_displayhook;
 
-        // The interpreter has the same scope as a `gil_scoped_release` instance
+        // Set m_release_gil_at_startup to true if
+        // the interpreter has the same scope as a `gil_scoped_release` instance
         // so that the GIL is not held by default, it will only be held when the
         // interpreter wants to execute Python code. This means that whenever
         // the interpreter will execute Python code it will need to create an
         // `gil_scoped_acquire` instance first.
         gil_scoped_release_ptr m_release_gil = nullptr;
+        bool m_release_gil_at_startup = true;
```

Comment:
Thanks for your PR! Would you mind adding a comment saying that the GIL **must** be released by the subclass if it's not released by `xpython::interpreter` (if `m_release_gil_at_startup` is false)?